convert_microns_to_nwb.ipynb

```diff
@@ -27,158 +25,162 @@
-    "        data=H5DataIO(movie.transpose(2, 0, 1), compression=True),\n",
+    "    # Transpose movie to first dimension be time\n",
+    "    movie = movie.transpose(2, 0, 1)\n",
+    "\n",
+    "    # Create optical series\n",
+    "    optical_series_kwargs = dict(\n",
+    "        name=\"visual_stimulus\",\n",
+    "        data=H5DataIO(movie, compression=True),\n",
     "        timestamps=H5DataIO(timestamps, compression=True),\n",
     "        unit=\"n.a.\",\n",
+    "        description=\"Stimulus movie\",\n",
+    "        distance=np.nan,  # unknown\n",
+    "        orientation=\"0 is up\",\n",
+    "        field_of_view=[np.nan, np.nan],\n",
     "    )\n",
-    "    \n",
+    "    optical_series = OpticalSeries(**optical_series_kwargs)\n",
     "    nwb.add_stimulus(optical_series)\n",
-    "    \n",
-    "    \n",
+    "\n",
+    "\n",
     "def add_eye_tracking(scan_key, nwb):\n",
-    "    pupil_min_r, pupil_maj_r, pupil_x, pupil_y, timestamps = (\n",
+    "\n",
+    "    pupil_x, pupil_y, timestamps = (\n",
     "        nda.RawManualPupil() & scan_key\n",
-    "    ).fetch1(\"pupil_min_r\", \"pupil_maj_r\", \"pupil_x\", \"pupil_y\", \"pupil_times\")\n",
-    "    \n",
+    "    ).fetch1(\"pupil_x\", \"pupil_y\", \"pupil_times\")\n",
+    "\n",
+    "    spatial_series_kwargs = dict(\n",
+    "        name=\"eye_position\",\n",
+    "        data=H5DataIO(np.c_[pupil_x, pupil_y], compression=True),\n",
+    "        timestamps=H5DataIO(timestamps, compression=True),\n",
+    "        unit=\"n.a.\",\n",
+    "        description=\"x,y position of eye\",\n",
+    "        reference_frame=\"unknown\",\n",
+    "    )\n",
+    "    eye_position = SpatialSeries(**spatial_series_kwargs)\n",
+    "\n",
+    "    eye_tracking = EyeTracking(eye_position)\n",
+    "    nwb.add_acquisition(eye_tracking)\n",
+    "\n",
+    "\n",
+    "def add_pupil_tracking(scan_key, nwb):\n",
+    "\n",
+    "    pupil_min_r, pupil_maj_r, timestamps = (\n",
+    "            nda.RawManualPupil() & scan_key\n",
+    "    ).fetch1(\"pupil_min_r\", \"pupil_maj_r\", \"pupil_times\")\n",
+    "\n",
     "    pupil_min_r = TimeSeries(\n",
     "        name=\"pupil_min_r\",\n",
     "        description=\"minor axis of pupil tracking ellipse\",\n",
     "        data=H5DataIO(pupil_min_r, compression=True),\n",
     "        timestamps=H5DataIO(timestamps, compression=True),\n",
-    "        unit=\"unknown\",\n",
+    "        unit=\"n.a.\",\n",
     "    )\n",
-    "    \n",
+    "\n",
     "    pupil_maj_r = TimeSeries(\n",
     "        name=\"pupil_maj_r\",\n",
-    "        description=\"jajor axis of pupil tracking ellipse\",\n",
+    "        description=\"major axis of pupil tracking ellipse\",\n",
     "        data=H5DataIO(pupil_maj_r, compression=True),\n",
-    "        timestamps=pupil_min_r,\n",
-    "        unit=\"unknown\",\n",
+    "        timestamps=H5DataIO(timestamps, compression=True),\n",
```

Comment:
Can we change the timestamps back? This is a PyNWB trick to reuse timestamps: https://pynwb.readthedocs.io/en/stable/tutorials/general/file.html#reusing-timestamps